src/CMakeLists.txt

```diff
@@ -3,6 +3,19 @@ set(CMAKE_CXX_FLAGS "${COLOBOT_CXX_FLAGS} ${MXE_CFLAGS}")
 set(CMAKE_CXX_FLAGS_RELEASE ${COLOBOT_CXX_FLAGS_RELEASE})
 set(CMAKE_CXX_FLAGS_DEBUG ${COLOBOT_CXX_FLAGS_DEBUG})
 
+# Groups source files for IDE project
+macro(group_sources SOURCES GROUP_ROOT)
+    foreach(SOURCE_FILE ${SOURCES})
+        get_filename_component(GROUP "${SOURCE_FILE}" PATH)
+        if(PLATFORM_WINDOWS)
+            string(REPLACE "/" "\\" GROUP "${GROUP}")
+            set(GROUP "${GROUP_ROOT}\\${GROUP}")
+        else()
+            set(GROUP "${GROUP_ROOT}/${GROUP}")
```

Comment:
I don't know if any other IDE cares about this setting, but if it does, it's going to expect path to be split by \\. Just use \\ always, independent from the system, like you did before 0b2ff9098e61aca87471dfbf9070d6d1b8c5047d.